float64 finger1_roll_joint_position
float64 finger1_prox_joint_effort
float64 finger1_dist_joint_effort
float64 finger2_roll_joint_position
float64 finger2_prox_joint_effort
float64 finger2_dist_joint_effort
float64 thumb_prox_joint_effort
float64 thumb_dist_joint_effort

# This might be better since we can set things explicitly
